### ISSUE TITLE
advice: Gaps in hillshading

### ISSUE THREAD

**Issue by robshep** [der-stefan/OpenTopoMap#166]
Hi,

I wonder if anybody knows why i'm seeing little gaps and missing edges in the hillshading?

![image](https://user-images.githubusercontent.com/1799763/40449784-7cc0ebee-5ed1-11e8-9591-4c3d93296041.png)

**Comment by guillaumebof** [comment]
i have the same bug on a fresh otm server installed following the how to for dummies.

**Comment by der-stefan** [comment]
Yes, those glitches come from the DEM. Here is some current information about NASADEM: https://gis.stackexchange.com/questions/267134/what-is-nasadem-and-when-it-will-be-released

**Comment by guillaumebof** [comment]
how does opentopomap.org succeed to remove those glitches ? by using another source of DEM files or by processing the viewfinderpanoramas ones ?

**Comment by guillaumebof** [comment]
@robshep no i don't succeed to remove the glitches... so i move back to openstreetmap for my project.
but i'm interested to know the workaround of the official opentopomap !

**Comment by der-stefan** [comment]
We use the DEM kindly provided by opensnowmap, not NASADEM yet.

**Comment by timautin** [comment]
@der-stefan The problem doesn't seem to come from the DEM. Indeed:

- when using the SRTM 1 arc second global DEM (the one used by OpenSnowMap), I also have the voids
- when I open my tifs in Photoshop, there are no voids

Sounds like a problem in the tif usage to me. Am I wrong?

@robshep Did you manage to fix the problem?

**Comment by timautin** [comment]
Allright, I found a solution, see https://github.com/der-stefan/OpenTopoMap/issues/185

**Comment by Laverdista668** [comment]
Hi all,

I have the same issue on my own tileserver built with the help of the How To from Stefan. Still the same after I changed the interpolation from bilinear to bicubic. To be sure that all tiles gonna be new rendered I deleted all the metatiles. Strangly I have still a hillshading after I deleted all the tiff from the ~/OpenTopoMap/Mapnik/dem Directory?!?

Any ideas how can I remove (or suppress) these annoying gaps and glitches? My dem are taken from http://srtm.csi.cgiar.org/srtmdata (these are processed datas and shall have no gaps)

Thank you in advanced

**Comment by der-stefan** [comment]
The new vector maps will most likely use the 30m Copernicus DEM (COP-30).
